config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,28 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
We have a Gem that enables us to configure the content security policy across our apps in one go: https://github.com/alphagov/govuk_app_config#content-security-policy-generation

We've already got a file in this repo for it https://github.com/alphagov/government-frontend/blob/master/config/initializers/csp.rb but it might be worth renaming that to `content_security_policy.rb` since that seems to be naming convention Rails is going for.